package.json

```diff
@@ -239,7 +239,8 @@
     "@metamask/network-controller@npm:^18.0.1": "patch:@metamask/network-controller@npm%3A18.0.1#~/.yarn/patches/@metamask-network-controller-npm-18.0.1-c4d0cfaecd.patch",
     "@metamask/network-controller@npm:^17.2.1": "patch:@metamask/network-controller@npm%3A18.0.1#~/.yarn/patches/@metamask-network-controller-npm-18.0.1-c4d0cfaecd.patch",
     "@metamask/keyring-controller@npm:^13.0.0": "patch:@metamask/keyring-controller@npm%3A13.0.0#~/.yarn/patches/@metamask-keyring-controller-npm-13.0.0-d94816a680.patch",
-    "@metamask/gas-fee-controller@npm:^14.0.1": "patch:@metamask/gas-fee-controller@npm%3A14.0.1#~/.yarn/patches/@metamask-gas-fee-controller-npm-14.0.1-59e9d16a4e.patch"
+    "@metamask/gas-fee-controller@npm:^14.0.1": "patch:@metamask/gas-fee-controller@npm%3A14.0.1#~/.yarn/patches/@metamask-gas-fee-controller-npm-14.0.1-59e9d16a4e.patch",
+    "@metamask/gas-fee-controller@npm:^15.1.0": "patch:@metamask/gas-fee-controller@npm%3A15.1.1#~/.yarn/patches/@metamask-gas-fee-controller-npm-15.1.1-f8009af091.patch"
```

Comment:
It looks like you're patching v15 here, but v14 is the version we're actually using

Reply:
Related:
- #24333

Reply:
but we are also using ^v15.1.0 because of this:

```
"@metamask/gas-fee-controller@npm:^15.1.0":
  version: 15.1.1
  resolution: "@metamask/gas-fee-controller@npm:15.1.1"
  dependencies:
    "@metamask/base-controller": "npm:^5.0.2"
    "@metamask/controller-utils": "npm:^9.1.0"
    "@metamask/eth-query": "npm:^4.0.0"
    "@metamask/ethjs-unit": "npm:^0.3.0"
    "@metamask/network-controller": "npm:^18.1.0"
    "@metamask/polling-controller": "npm:^6.0.2"
    "@metamask/utils": "npm:^8.3.0"
    "@types/bn.js": "npm:^5.1.5"
    "@types/uuid": "npm:^8.3.0"
    bn.js: "npm:^5.2.1"
    uuid: "npm:^8.3.2"
  peerDependencies:
    "@metamask/network-controller": ^18.0.0
  checksum: 6e0ddf10d4dde13e4da8bb17e7503a68f64369e51958059168f3400941b540dd02df417e60451a5bdb2be9d9fb1d4a11ae9685f811b0a4f1744ff23ce1c7b1ff
  languageName: node
  linkType: hard
```

Reply:
> but we are also using ^v15.1.0 because of this:

If you follow the dependency chain, you see this is pulled in from `@metamask/transaction-controller` and `@metamask/user-operation-controller`, which only pull it in for types. The patched file for this resolution is not used (but would be after #24333).

Reply:
oh...

Reply:
Posted a comment below https://github.com/MetaMask/metamask-extension/pull/24416#issuecomment-2099293047

> There is an existing patch for the actually used version on the line above and to make this new patch be effective, it should be added to @metamask-gas-fee-controller-npm-14.0.1-59e9d16a4e.patch.

This would have no effect, because the need for the patch is only in the 15.1.0 and up code. So I shouldn't have even created this PR in the first place